<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-I63MURL" name="org.example.testRunner.Runners" tests="15" failures="2" timestamp="2022-07-20T00:26:59 EET" time="476.807" errors="1">
  <testcase name="runScenario" time="27.024" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="14.661" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="12.879" classname="org.example.testRunner.Runners">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at org.example.stepDefs.D06_homeSlidersStepDef.assertUrlSlider1(D06_homeSlidersStepDef.java:22)
at ✽.assert the url of first slider(file:///C:/Users/jack/IdeaProjects/demonopcommerce/src/main/resources/features/F06_homeSliders.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="14.103" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="19.014" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="66.369" classname="org.example.testRunner.Runners">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at org.example.stepDefs.D06_homeSlidersStepDef.assertUrlSlider2(D06_homeSlidersStepDef.java:42)
at ✽.assert the url of second slider(file:///C:/Users/jack/IdeaProjects/demonopcommerce/src/main/resources/features/F06_homeSliders.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="50.395" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="13.998" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="13.012" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="24.311" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="43.814" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="73.891" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="55.929" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="20.690" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="26.717" classname="org.example.testRunner.Runners">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at org.example.stepDefs.D07_followUsStepDef.assertCurrentUrl(D07_followUsStepDef.java:27)
at ✽.assert current url "https://demo.nopcommerce.com/new-online-store-is-open"(file:///C:/Users/jack/IdeaProjects/demonopcommerce/src/main/resources/features/F07_followUs.feature:13)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- org.example.testRunner.Runners -->
